lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -49,6 +49,16 @@ static Value getOrInsertDevice(ConversionPatternRewriter &rewriter,
   return deviceOp.getResult();
 }
 
+template <typename OpAdaptor>
+static void removeDpsOp(ConversionPatternRewriter &rewriter,
```

Comment:
Rename to `removeDestination` or `undoDps` or something like that.

Reply:
Nit: this name kinda sounds like it's removing the op itself.  Maybe `removeDpsEmptyOp`.  Also are we just missing some traits that would auto-cleanup the empty's?  It seems like it might be unnecessary to manually erase them.

Reply:
Didn't know about this trait. Added tnx nick